daemons/based/based_callbacks.c

```diff
@@ -1440,6 +1440,23 @@ cib_process_command(xmlNode *request, const cib__operation_t *operation,
             cib_read_config(config_hash, result_cib);
         }
 
+        /* @COMPAT Nodes older than feature set 3.19.0 don't support
+         * transactions. In a mixed-version cluster with nodes <3.19.0, we must
+         * sync the updated CIB, so that the older nodes receive the changes.
+         * Any node that has already applied the transaction will ignore the
+         * synced CIB.
+         *
+         * To ensure the updated CIB is synced from only one node, we sync it
+         * from the originator.
+         */
+        if ((operation->type == cib__op_commit_transact)
+            && pcmk__str_eq(originator, OUR_NODENAME, pcmk__str_casei)
```

Comment:
`cib_peer_callback()` also uses `pcmk__str_null_matches` but that doesn't look appropriate, at least for this case.